core/google/api/core/operations_v1/operations_client_config.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""gapic configuration for the googe.longrunning.operations client."""
+
+config = {
+    "interfaces": {
+        "google.longrunning.Operations": {
+            "retry_codes": {
+                "idempotent": [
+                    "DEADLINE_EXCEEDED",
+                    "UNAVAILABLE"
+                ],
+                "non_idempotent": []
+            },
+            "retry_params": {
+                "default": {
+                    "initial_retry_delay_millis": 100,
+                    "retry_delay_multiplier": 1.3,
```

Comment:
`1.3` is not "exactly representable" in binary64. Why not `1.3125`? Or who cares is also appropriate

Reply:
I don't know what you mean here.

Reply:
There is no such thing as `1.3` in binary because you can't divide by `5` in binary without having a repeating binary expansion. In binary, it's

```
1.3 = 1.0(1001)(1001)(1001)(1001)(1001)...._2
```

so the number has to be rounded to be put into `binary64`.

```
1.3125 = 1.0101_2
```

so no rounding needed.

Reply:
I don't think we should worry about that right now. :)

Reply:
Right. Mine was more a warning: there is no such thing as `1.3` on today's computers, so use exactly representable numbers when possible.

Reply:
Because the world will end if it gets rounded to something close enough?

Reply:
No. That's the beauty of our floating point system. I was just saying "don't expect it to be what you asked for". It's worth knowing, no?